plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/glue/TestHiveGlueMetastore.java

```diff
@@ -1332,6 +1332,13 @@ public void testTableWithoutStorageDescriptor()
         }
     }
 
+    @Test
+    public void testGetAllEntitiesOnAccessDeniedDatabase()
+    {
+        assertThat(metastore.getAllTables("forbidden")).isEmpty();
```

Comment:
Comments are currently missing because the PR is still in draft mode and represents just a demo on how to test the reported limitation https://github.com/trinodb/trino/issues/14746 on hive backed by Glue metastore.